tests/test_attributes.py

```diff
@@ -11,7 +11,7 @@
 
 from .testutils import PARAMETRIZE_CONTEXT_BEHAVIOR, setup_test_config
 
-setup_test_config({"autodiscover": False})
```

Comment:
Unrelated, but I modified `setup_test_config` to set autodiscovery to False by default in tests, so I could remove a lot of these.